config/core/test-configs.yaml

```diff
@@ -2622,6 +2622,12 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-rtc
```

Comment:
Failed to run with Clang: https://staging.kernelci.org/test/plan/id/62656feb936378c0e2178696/

Some test cases also failed with GCC: https://staging.kernelci.org/test/plan/id/62666a380de131e7da17869e/